samples/java-customer-registry-kafka-quickstart/README.md

```diff
@@ -0,0 +1,86 @@
+# Quickstart project: Customer Registry
+
+## Designing
+
+To understand the Akka Serverless concepts that are the basis for this example, see [Designing services](https://developer.lightbend.com/docs/akka-serverless/services/development-process.html) in the documentation.
+
+
+## Developing
+
+This project demonstrates the use of Value Entity and View components.
+To understand more about these components, see [Developing services](https://developer.lightbend.com/docs/akka-serverless/developing/index.html)
+and in particular the [Java section](https://developer.lightbend.com/docs/akka-serverless/java-services/index.html)
+
+
+## Building
+
+You can use Maven to build your project, which will also take care of
+generating code based on the `.proto` definitions:
+
+```
+mvn compile
+```
+
+
+## Running Locally
+
+To run the example locally, you must run the Akka Serverless proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+
+### with Google Pub/Sub
```

Comment:
Should we remove this section since it's all about Kafka integration?

Reply:
Actually, I added the possibility to run it locally.

Reply:
I would prefer if it was focused on one thing, Kafka.

Reply:
I see. Ok.